db/seeds.rb

```diff
@@ -0,0 +1,46 @@
+melanie = User.create(username: 'melanie', password: 'password', email: 'melaniegzmail.com')
+sherianne = User.create(username: 'sherianne', password: 'password', email: 'sherianne@gmail.com')
+allain = User.create(username: 'allain', password: 'password', email: 'allain@j.com')
+jenna = User.create(username: 'jenna', password: 'password', email: 'jenna@gmail.com')
+jung = User.create(username: 'jung', password: 'password', email: 'jung@gmail.com')
+natalie = User.create(username: 'natalie', password: 'password', email: 'natalie@gmail.com')
+
+dog = Question.create(title: 'Dog question', body: "Do you like dogs?", user: melanie)
+cat = Question.create(title: 'Cat question', body: "Is it ok to have a cat in this apartment?", user: jenna)
+hamster = Question.create(title: 'Hamster question', body: "Does your hamster like to eat mini pies?", user: allain)
+monkey = Question.create(title: 'Monkey Question', body: "What do monkeys eat?", user: natalie)
+hippo = Question.create(title: 'Hippo Question', body: "Are hippos friendly?", user: jung)
+
+answer_1 = Answer.create(body: 'duh, how could you not!', question: dog, user: jung)
+answer_2 = Answer.create(body: 'As long as it will not scratch me', question: cat, user: natalie)
+Answer.create(body: 'Always!', question: hamster)
+Answer.create(body: 'yes!!!', question: cat)
+answer_3 = Answer.create(body: 'bananas...', question: monkey, user: jenna)
+answer_4 = Answer.create(body: 'never', question: hippo, user: allain)
+
+comment_1 = Comment.create(body: 'Dolore illum animi et neque accusantium.'commenter_id: 2, commentable: cat, )
```

Comment:
we don't have a commenter defined anywhere, i think you want user_id. unless we change our migration and corresponding model associations to refer to the user as a commenter. pulling in @jenvanc for consideration.